src/components/data-grid/RowActionsCell.tsx

```diff
@@ -25,7 +25,7 @@ export const RowActionsCell = ({ data }: RowActionsCellProps) => {
   };
 
   const onEdit = () => {
-    data.onEdit(data.id);
+    data.onEdit(data.contactId);
```

Comment:
Problem is that now it isn't generic anymore

I think it would be better to pass the entire data object
```suggestion
    data.onEdit(data);
```
and then handle it as follows in `Caesar`

```
onEdit={({ contactId }) => { // Do something }}
```

Reply:
@hiddeottens yeah, agrred. I've updated the PR